packages/slate-hyperscript/src/creators.ts

```diff
@@ -24,6 +24,17 @@ import {
 
 const STRINGS: WeakSet<Text> = new WeakSet()
 
+function flattenDescendants(array: any[]) {
```

Comment:
Instead of doing this, traditionally we've recommended that consumers of slate and it's peripheral libraries use polyfills to get access to calls that aren't available on dead browsers